### PR TITLE
retire license category reporting

### PR DESCRIPTION
**Pull request by iamsid2**
Tried to solve issue[106](https://github.com/nexB/deltacode/issues/106)
Signed-off-by: iamsid2 <shaktimund97@gmail.com>

### CODE REVIEW

File: src/deltacode/utils.py [hunk 1]
```diff
@@ -48,70 +48,71 @@ def update_from_license_info(delta, unique_categories):
...
+        delta.update(score=15, factor='license info removed')
         return
 
     new_licenses = delta.new_file.licenses or []
     old_licenses = delta.old_file.licenses or []
 
-    new_categories = set(license.category for license in new_licenses)
-    old_categories = set(license.category for license in old_licenses)
+    # new_categories = set(license.category for license in new_licenses)
+    # old_categories = set(license.category for license in old_licenses)
```
Comment (steven-esser) [inline src/deltacode/utils.py]: Delete these lines instead of adding comments

File: src/deltacode/utils.py [hunk 2]
```diff
@@ -48,70 +48,71 @@ def update_from_license_info(delta, unique_categories):
...
-        return
+        # delta.update(20, 'license info added')
+        #
+        # for category in new_categories:
+        #     # no license ==> 'Copyleft Limited'or higher
+        #     if category in unique_categories:
+        #         delta.update(20, category.lower() + ' added')
+        #     # no license ==> 'Permissive' or 'Public Domain'
+        #     else:
+        #         delta.update(0, category.lower() + ' added')
```
Comment (steven-esser) [inline src/deltacode/utils.py]: Same thing here, see above comments

File: src/deltacode/utils.py [hunk 3]
```diff
@@ -48,70 +48,71 @@ def update_from_license_info(delta, unique_categories):
...
+        #     unique_categories_in_old_file = len(old_categories & unique_categories)
+        #     # 'Permissive' or 'Public Domain' ==> 'Copyleft Limited' or higher
+        #     if unique_categories_in_old_file == 0 and category in unique_categories:
+        #         delta.update(20, category.lower() + ' added')
+        #     # at least 1 category in the old file was 'Copyleft Limited' or higher ==> 'Copyleft Limited' or higher
+        #     elif unique_categories_in_old_file != 0 and category in unique_categories:
+        #         delta.update(10, category.lower() + ' added')
+        #     # 'Permissive' or 'Public Domain' ==> 'Permissive' or 'Public Domain' if not in old_categories
+        #     elif category not in unique_categories:
+        #         delta.update(0, category.lower() + ' added')
```
Comment (steven-esser) [inline src/deltacode/utils.py]: Same comment as above

**Review by steven-esser — CHANGES_REQUESTED** [review]
You have a number of failing tests. Please fix these. 

You can run the test suite locally via `py.test`.

**Comment by iamsid2** [comment]
yeah working on it now.

**Comment by iamsid2** [comment]
Looks like there is a clash between python 3.6 and 2.7 as i have anaconda installed which as python 3.6 as its root

**Comment by steven-esser** [comment]
@iamsid2 Currently, DeltaCode uses python2.7. If you install python2.7, it can exist alongside python36 and our configure scripts will be smart enough to use the correct python version.

**Comment by steven-esser** [comment]
Closing due to stagnation.

**Pull request closed without merging**